samples/nfc/tnep_tag/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+# Check if selected board is supported.
+if(${BOARD} STREQUAL "nrf52_pca10040")
```

Comment:
internal nrf boilerplate has `NRF_SUPPORTED_BOARDS`
see `nrf/cmake/boilerplate.cmake`

Reply:
Good point. Switched to using it